package.json

```diff
@@ -22,6 +22,9 @@
     "webpack-dev-server": "^4.10.0"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
přidávala sis závislost na font awesome ale nikde je nevidím použité, to byl záměr?

Reply:
Jejda, ja jsem ten font awesome ikonku chtela naimportovat pres ty zavislosti, ale pak mi to nefungovalo, tak jsem ji stahla jako svg a pouzila primo. No a zavislost jsem zapomnela vymazat.